apps/greenhouse/src/components/nav/PluginNav.tsx

```diff
@@ -69,8 +70,13 @@ const PluginNav = () => {
   const { data: authData, loggedIn, login, logout } = useAuth()
 
   const navigateToApp = (appId: string) => {
+    // save the url state of the current app
```

Comment:
Does the comment mean to tell us that?
`// Save the current app's URL state to restore it later`

If so, can you adjust it?